micrometer-core/src/test/java/io/micrometer/core/instrument/simple/SimpleMeterRegistryTest.java

```diff
@@ -101,6 +109,133 @@ public void newFunctionCounterWhenCountingModeIsStepShouldReturnStepFunctionCoun
         assertThat(functionCounter).isInstanceOf(StepFunctionCounter.class);
     }
 
+    @Test
+    public void toStringOutputTest() {
+        final SimpleMeterRegistry registry = new SimpleMeterRegistry();
+
+        final Counter requestSizeCounter = Counter.builder("request.size")
+                .baseUnit("bytes")
+                .register(registry);
+
+        final DistributionSummary responseSizeSummary = DistributionSummary.builder("response.size")
+                .baseUnit("bytes")
+                .publishPercentileHistogram()
+                .publishPercentiles(0.9999, 0.99, 0.95, 0.90, 0.75)
+                .sla(250, 500, 1000, 2000, 4000, 6000)
+                .register(registry);
+
+        final Timer responseTimer = Timer.builder("response.time")
+                .tag("action", "create")
+                .publishPercentiles(0.999, 0.99, 0.95, 0.90, 0.75)
+                .sla(Duration.ofMillis(20), Duration.ofMillis(50), Duration.ofMillis(100))
+                .publishPercentileHistogram()
+                .register(registry);
+
+        final AtomicLong testRunTime = new AtomicLong();
+        TimeGauge
+                .builder("test.run.time", testRunTime, TimeUnit.NANOSECONDS, AtomicLong::doubleValue)
+                .register(registry);
+
+        FunctionTimer
+                .builder("cache.latency", new Object(), o -> 10, o -> 300, TimeUnit.MILLISECONDS)
+                .register(registry);
+
+        FunctionCounter
+                .builder("cache.evictions", new Object(), o -> 10)
+                .baseUnit("beans")
+                .register(registry);
+
+        final LongTaskTimer taskTimer = LongTaskTimer
+                .builder("test.loop.timer")
+                .register(registry);
+
+        Meter.builder("default.meter", Type.GAUGE,
+                Arrays.asList(
+                        new Measurement(() -> 7.7, Statistic.VALUE),
+                        new Measurement(() -> 8.8, Statistic.VALUE)
+                ))
+                .baseUnit("beans")
+                .register(registry);
+
+        final LongTaskTimer.Sample loopSample = taskTimer.start();
+
+        for (int i = 0; i < 300; i++) {
+            responseTimer.record(Duration.ofMillis(i));
+            requestSizeCounter.increment(i);
+            responseSizeSummary.record(i);
+        }
+
+        loopSample.stop();
+
+        testRunTime.set(200);
+        assertThat(registry.toString()).isEqualTo("\n"
+                + "CumulativeFunctionCounter cache.evictions count=10.000000\n"
```

Comment:
Instead of showing the meter class name this way. It is more useful to just know the type (`counter`, `gauge`, etc)